code/addons/a11y/src/components/VisionSimulator.tsx

```diff
@@ -144,7 +145,7 @@ export const VisionSimulator = () => {
         onDoubleClick={() => setFilter(null)}
       >
         <IconButton key="filter" active={!!filter} title="Vision simulator">
-          <Icons icon="accessibility" />
+          <Icon.Accessibility />
```

Comment:
According to the docs you wrote:
> The meaning can be obvious visually, but it must have a proper text alternative via `aria-label` for screen readers

Should you make sure every icon in this PR has the proper aria-label applied to it?

Reply:
You're right I could spend some time to do that. That could be a good accessibility win.

Reply:
@yannbf That would be great but that would require quite some time to test everything. I would put that effort in a separate PR though with other efforts to improve accessibility.

Reply:
I found a new way to help accessibility in a semi automated way @yannbf. I'll explore this more in a separate PR.